gps/index.html

```diff
@@ -10,27 +10,27 @@
 date: 2018-04-09
 categories: []
 ---
-<h2 class="text-left"><i class="fas fa-fw fa-question"></i> What is a Reference Network</h2>
+<h2 class="text-left"><i class="fas fa-fw fa-question"></i> What Is a Reference Network?</h2>
```

Comment:
So here, I've used headline-style capitalization, which is what I typically use for AGRC stuff, but as I've mentioned before, if you don't want to follow that, that's fine! Just be consistent with whatever you choose.

Reply:
I am ok with this style. No need to point it out.

Reply:
Good good :) Added to the AGRC style sheet (still in the works).